it/content/src/main/content/jcr_root/content/dam/formsanddocuments/core-components-it/samples/imagetestgroup/.content.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Inside samples create a replace folder, inside replace folder create collateral based on the use-case, for example, in this case, the name of the form would be `replacewcmcomponents`